pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -144,375 +127,6 @@ func init() {
 	InfraSAs.saToRole = map[string]authorizationapi.ClusterRole{}
 
 	err = InfraSAs.addServiceAccount(
-		InfraPersistentVolumeRecyclerControllerServiceAccountName,
```

Comment:
everything added here gets auto-reconciled on server start. moving these broke upgrades for controllers that cause server exits if they are missing permissions (because the ansible reconciliation can't run until the server is up). see https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin/3303/testReport/junit/(root)/Reconcile%20Cluster%20Roles%20and%20Cluster%20Role%20Bindings%20and%20Security%20Context%20Constraints/_localhost__Reconcile_Cluster_Roles/